keyboards/handwired/polly40/.vscode/settings.json

```diff
@@ -0,0 +1,3 @@
+{
```

Comment:
This should not be included.

Reply:
I removed it already. it was from my ide